src/components/utils.js

```diff
@@ -31,3 +35,36 @@ export const createReviewCards = (reviewData) => {
 		);
 	});
 };
+
+export const createSingleReview = (review) => {
```

Comment:
this shouldn't need to be exported into a separate function as this is what the separate components are for. you are essentially adding another layer of extraction which doesn't need to be here - this can go in your SingleReview component so it's in one place and easy to read